assembly/src/parsers/context.rs

```diff
@@ -279,7 +317,8 @@ impl ParserContext {
             docs,
             num_locals,
             is_export,
-            body,
+            body: nodes,
+            locations,
```

Comment:
We can probably replace these with `body: ParsedCode`.

Reply:
Done in https://github.com/0xPolygonMiden/miden-vm/pull/885/commits/22abec980a1edbf1d84541cd369fff26008ea96b